### DISCUSSION TITLE
Merge columns

### DISCUSSION THREAD

**Discussion by DZ4-Miguel** [Power-Components/livewire-powergrid#742] [Q&A] [answered]
Hello,

is it possilble to join two columns together as in my example? 

I want this: 
![image](https://user-images.githubusercontent.com/107668723/204753351-bac5c893-ee35-48ef-9a79-d096bafb5b7a.png)

become like this: 

![image](https://user-images.githubusercontent.com/107668723/204753468-746be026-241c-4419-a0b0-e2a203338697.png)

thank you.

**Comment by BenOussama180** [comment, marked as answer]
just put it in the closure ? 
`->addColumn('full_name', function (Model $model) {
                return $model->name . ' ' . $model->first_name;
            })`
